Src/Logic/LoginButton/LoginButton.ts

```diff
@@ -2,23 +2,26 @@ import {readFromLocalStorage} from '../LocalStorage/LocalStorage';
 import {User} from '../../../Models/User.model';
 import {setLoggedInUser, getLoggedInUser} from '../../UIComponents/utils/utils';
 import {USER_INPUT_EMPTY} from '../../../Constants/consts';
-function loginButton(username: string, userDashboard: (user: User) => HTMLElement, failComp: HTMLParagraphElement): void {
+import showModalWindow from '../../../Src/UIComponents/ModalWindow/ModalWindow';
+function loginButton(username: string, userDashboard: (user: User) => HTMLElement): void {
+     
     const loggedIn = getLoggedInUser();
     const user = readFromLocalStorage(loggedIn ? loggedIn : username) as User | null;
     const loginBtn = this as HTMLButtonElement;
```

Comment:
In aim to fix my bug replace these lines
```
    const loggedIn = getLoggedInUser();
    const user = readFromLocalStorage(loggedIn ? loggedIn : username) as User | null;
    const loginBtn = this as HTMLButtonElement;
```
by this line
`const user = readFromLocalStorage(username) as User | null;`

Reply:
Done